feat: update docs with missing parts (#564)
vegaprotocol/documentation: docs/node-operators/migration-guides/upgrade-to-0702.md

There is no change required in the `tendermint` config. If you have to prepare the `tendermint` config from scratch, use the instruction for [v0.68.2](./upgrade-to-0682.md#8-update-tendermint-config).

### 9. Migrate tendermint data when using default home path

:::warning
You do not need to execute this step when providing the custom `--home` flat for the tendermint. This step MUST be executed when the default tendermint home directory path is used.
:::

> The brief background is, We have migrated the Tendermint to the [CometBFT](https://github.com/cometbft/cometbft), which is the fork of tendermint. 
> 
> This change implies a change of the default home directory. If you have not provided the `--home` flag for the tendermint, it determined the default home in the `<USER_HOME>/.tendermint`. Now you should migrate your data into the `<USER_HOME>/.cometbft` directory. You have at least two options to do it:

Reviewer: ```suggestion
The Vega implementation of Tendermint has been migrated to using [CometBFT](https://github.com/cometbft/cometbft), which is a fork of Tendermint. 

This implies a change to the default home directory. If you have not provided the `--home` flag for Tendermint, it determines the default home in the `<USER_HOME>/.tendermint`. Now, you should migrate your data into the `<USER_HOME>/.cometbft` directory. You have at least two options for how to do it:
```
